Yoast/wordpress-seo
# Update HowTo block's functionality and JSON-LD

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Only one HowTo/FAQ block is now allowed per page. The Gutenberg editor reflex this by not suggesting either block when it is used, and by greying out the icon in the new block drop-down.
* Images only work in the HowToStep (accompanying) text, so not the step name. Same for the FAQ (only in answers, not in questions).

## Test instructions

This PR can be tested by following these steps:

* If applicable to you, make sure you have the feature flag in the wp-config:
`define('YOAST_FEATURE_GUTENBERG_STRUCTURED_DATA_BLOCKS', true );`
* Make a post, and try to add as many HowTo / FAQ blocks as you can. Should only be able to add 1 per type.
* Fill out the How To block (add some images too), and save the page.
* In the frontend, inspect the html, and locate the `<script type="application/ld+json">`. Copy its contents, and place them in [SDTT](https://search.google.com/structured-data/testing-tool/u/0/) as a code fragment. Verify that it looks as in the issue.

## Quality assurance


Fixes #10706
